frontend/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import FoxticketLogo from '../../assets/images/logos/Foxticket.png'
+import Logout from './Logout';
+import { Link, BrowserRouter } from "react-router-dom";
+import { connect } from 'react-redux';
+import './Header.styles.sass'
+
+function Header(props) {
+
+  return (
+    <>
+    <nav className="NavbarItems" >
+      <img src={FoxticketLogo} alt='navigation bar logo'></img>
+      <h3 className="nav-text">Foxticket</h3>
+        <ul className="nav-menu">
+        { 
+          props.type ==='admin' &&
+          <BrowserRouter>
```

Comment:
same here, this is definitely not good, use MemoryRouter for testing

https://reactrouter.com/web/guides/testing

Reply:
Again: the entire app is wrapped in a Router, you should mock this in tests

Reply:
well, yes.. testing is not my strength :D So I tried to succeed all of them this way, and if nothing else is broken , remove these lines and make it work in a normal way , just between the 2 stages I pushed